plugins/woocommerce/src/Blocks/BlockTypes/MiniCart.php

```diff
@@ -498,13 +499,19 @@ protected function render_experimental_iapi_mini_cart( $attributes, $content, $b
 			wp_interactivity_config(
 				$this->get_full_block_name(),
 				array(
-					'addToCartBehaviour' => $attributes['addToCartBehaviour'],
+					'addToCartBehaviour'   => $attributes['addToCartBehaviour'],
+					'onCartClickBehaviour' => $on_cart_click_behaviour,
+					'checkoutUrl'          => wc_get_checkout_url(),
 				)
 			);
 
 			$cart_always_shows_price = isset( $attributes['hasHiddenPrice'] ) && false === $attributes['hasHiddenPrice'];
 			$price_color             = isset( $attributes['priceColor']['color'] ) ? $attributes['priceColor']['color'] : '';
 
+			$button_role = 'open_drawer' === $on_cart_click_behaviour
+				? ''
+				: 'role="link"';
```

Comment:
We might want to change this conditional as well to check `navigate_to_checkout`.

Reply:
🤦🏻 good point.